### ISSUE TITLE
Multiple entries, lazy loading and duplicates

### ISSUE THREAD

**Issue by mstrelex** [kisenka/svg-sprite-loader#52]
I have multiple entries (chunks) that are lazy loaded in an AngularJS app.
When chunk is loaded, DOMContentLoaded is fired and SVGs are added again to DOM, creating duplicates. Can this somehow be avoided ?

**Comment by princed** [comment]
It's definitely a bug and should be fixed.

**Comment by kisenka** [comment]
Duplicate of #8

**Comment by kisenka** [comment]
@mstrelex in [new version](https://github.com/kisenka/svg-sprite-loader/releases/tag/v2.0.1) sprite renders only once. You can check corresponding [test](https://github.com/kisenka/svg-baker/blob/6449bbac0e9c00e75ef99c8b513d44926ac5292a/packages/svg-baker-runtime/test/browser-sprite.test.js#L141).
